enterprise/reporting/reporting.go

```diff
@@ -556,6 +556,9 @@ func (r *DefaultReporter) Report(metrics []*types.PUReportedMetric, txn *Tx) err
 		return fmt.Errorf("preparing statement: %v", err)
 	}
 	defer func() { _ = stmt.Close() }()
+	txn.AddSuccessListener(func() {
+		NewEventStatsReporter(r.configSubscriber, stats.Default).Report(metrics)
+	})
```

Comment:
We can probably create the `EventStatsReporter` once and use it here?

```suggestion
	txn.AddSuccessListener(func() {
		r.statsReporter.Report(metrics)
	})
```
```
type DefaultReporter struct {
    .
    .
    .
	requestLatency            stats.Measurement
}

func NewDefaultReporter(statsFactory stats.Stats, ctx context.Context, log logger.Logger, configSubscriber *configSubscriber) *DefaultReporter {
	return &DefaultReporter{
		.
        .
        .
		statsReporter:                        NewEventStatsReporter(configSubscriber, statsFactory),
	}
}
```